Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN apt-get update && \
         php5-curl=5.6.7+dfsg-1+deb.sury.org~trusty+1 \
         php5-gearman=1.1.2-1+deb.sury.org~trusty+2 \
         php5-memcache=3.0.8-5+deb.sury.org~trusty+1 \
+        php5-memcached=2.2.0-2+deb.sury.org~trusty+1 \
```

Comment:
eventually would move entirely to this, but need to support legacy in the interim